buildspec/merge_master.yaml

```diff
@@ -3,12 +3,14 @@ phases:
   pre_build:
     commands:
       - echo Logging in to Amazon ECR...
-      - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
+      - aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin ${REPOSITORY_URI}
```

Comment:
small thing.  can you change REPOSITORY_URI to FIPS_REPOSITORY_URI so we can use the fips modules for codebuild?